autoendpoint/src/routers/apns/router.rs

```diff
@@ -202,7 +202,9 @@ impl Router for ApnsRouter {
         router_input: &RouterDataInput,
         app_id: &str,
     ) -> Result<HashMap<String, Value>, RouterError> {
-        if !self.clients.contains_key(app_id) {
+        let quotes: &[_] = &['"', '\''];
+        let mapp = app_id.trim_matches(quotes);
```

Comment:
I think we've ruled the quoting out completely so we might as well remove all these trim calls.